utils/browser/src/lib.rs

```diff
@@ -151,12 +152,11 @@ pub fn start_client(mut task_manager: TaskManager, rpc_handlers: RpcHandlers) ->
 			Box::pin(async move {
 				let _ = task_manager.future().await;
 			}),
-		).map(drop)
+		)
+		.map(drop),
```

Comment:
Just curious, can this newline be disabled?  I donno substrate's exiting style though.

Reply:
overflow_delimited_expr can help sometimes in these situations but on balence I think the code works better left to the default value.

Reply:
I think the original is indeed better readable here.
Ideally would be a rule to:
- keep `(..).map(drop),` as such
- while letting `(..).map(drop).foo(bar)` gets split eventually, whether always or dependending on the line width (I would be fine with both options eitherway)